merino/jobs/navigational_suggestions/utils.py

```diff
@@ -95,3 +97,27 @@ def update_top_picks_with_firefox_favicons(
         firefox_favicon: Optional[str] = FIREFOX_PACKAGED_FAVICONS.get(domain)
         if firefox_favicon:
             domain_metadata["icon"] = firefox_favicon
+
+
+def _blocklist_path() -> str:
+    return os.path.join(
+        os.path.dirname(__file__),
+        "data/domain_blocklist.json",
+    )
+
+
+def load_blocklist(path: Optional[str] = None) -> set[str]:
+    """Load the blocklist into a set."""
+    if path is None:
+        path = _blocklist_path()
+    with open(path, "r") as fp:
+        domains = json.load(fp)
+        return {domain for domain in domains}
+
+
+def write_blocklist(blocklist: set[str], path: Optional[str] = None):
+    """Write the blocklist set to a file."""
+    if path is None:
+        path = _blocklist_path()
```

Comment:
```suggestion
def write_blocklist(blocklist: set[str], path: str = _blocklist_path()):
    """Write the blocklist set to a file.""
```

Probably, same here?